cli/flox/doc/flox.md

```diff
@@ -1,76 +1,33 @@
-% FLOX(1) Flox User Manuals
+---
+title: FLOX
+section: 1
+header: "Flox User Manuals"
+...
 
 # NAME
 
 flox - command-line interface (CLI)
 
 # SYNOPSIS
 
-flox [ `<options>` ] command [ `<command options>` ] [ `<args>` ] ...
-
-# DESCRIPTION
-
-flox is a platform for developing, building, and using packages created with Nix.
-It can be used alone to simplify the process of working with Nix,
-within a team for the sharing of development environments,
-and in enterprises as system development lifecycle (SDLC) framework.
-
-The `flox` CLI is used:
-
-1. To manage and use collections of packages,
-   environment variables and services
-   known as flox *runtime environments*,
-   which can be used in a variety of contexts
-   on any Linux distribution, in or out of a container.
-1. To launch flox *development environments*
-   as maintained using a `flox.nix` file
-   stored within a project directory.
-1. As a wrapper for Nix functionality
-   which drives the process of building packages with flox.
-
-<!--
-See *floxtutorial(7)* to get started.
-More in-depth information is available by way of the [flox User's Manual](https://alpha.floxsdlc.com/docs/).
--->
-
-## Command Line Completions
-
-Flox ships with command line completions for `bash`, `zsh` and `fish`.
-To enable completions in your shell Add the following to your shell's
-startup script:
-
-**When using the flox installers**
-
-- `bash`, `zsh`
-```
-export XDG_DATA_DIRS="/usr/local/share:$XDG_DATA_DIRS"
-```
-
-- `fish`
 ```
-set -px XDG_DATA_DIRS "/usr/local/share"
+flox [<general options>] <command>
+     [<command options>]
+     [<args>] ...
 ```
 
-**When installing flox through Nix**
-
-- `bash`, `zsh`
-```
-# for single user user installs
-export XDG_DATA_DIRS="$HOME/.nix-profile/share:$XDG_DATA_DIRS"
+# DESCRIPTION
 
-# for global installs
-export XDG_DATA_DIRS="/nix/var/nix/profiles/default:$XDG_DATA_DIRS"
+Flox is a virtual environment and package manager all in one.
 
-```
+With flox you create environments that layer and provide dependencies just
+where it matters,
+making them portable across the full software lifecycle.
 
-- `fish`
-```
-# for single user user installs
-set -px XDG_DATA_DIRS "$HOME/.nix-profile/share"
+## Command Line Completions
 
-# for global installs
-set -px XDG_DATA_DIRS "/nix/var/nix/profiles/default"
-```
+Flox ships with command line completions for `bash`, `zsh` and `fish`.
+These completions are installed alongside Flox.
```

Comment:
nonblocking question: does this all work out of the box?

Reply:
I tried it on Bash on Ubuntu and it worked. You can also see that we're installing completions here: https://github.com/flox/flox/blob/f75e3096730f91cbdb544b1e3bc11ce5f07abd9f/pkgs/flox-cli/default.nix#L174